source/services/wix/extractors/media-manager/index.js

```diff
@@ -82,6 +85,7 @@ module.exports = {
 				} );
 			} else {
 				wxr.addPost( {
+					id: IdFactory.get( file.original_file_name ),
```

Comment:
I think this needs to be changed, it's resulting in duplicates in the WXR.

Reply:
Minus an example this is hard to judge. I could see the duplicates come from the fact that an image is added to the WXR from static pages and at the same time the image resides in the media library. I've added a commit that tries to avoid these duplicates; there was an `exists()` check in static images, this adds another check here.